src/settings.cpp

```diff
@@ -624,29 +624,36 @@ void read_settings_xml(pugi::xml_node root)
   }
 
   // Shannon Entropy mesh
-  if (check_for_node(root, "entropy_mesh")) {
-    int temp = std::stoi(get_node_value(root, "entropy_mesh"));
-    if (model::mesh_map.find(temp) == model::mesh_map.end()) {
-      fatal_error(fmt::format(
-        "Mesh {} specified for Shannon entropy does not exist.", temp));
+  if (solver_type == SolverType::RANDOM_RAY) {
```

Comment:
Might change the `// Shannon Entropy mesh` comment to just `// Shannon Entropy` as this section is doing more now.

Reply:
Changed, thanks!